spatialmath/geometry.go

```diff
@@ -28,6 +28,7 @@ type Geometry interface {
 	DistanceFrom(Geometry) (float64, error)
 	EncompassedBy(Geometry) (bool, error)
 	Label() string
+	ToPointCloud(float64) []r3.Vector
```

Comment:
I feel unsure if this is the right name for this method. This is because the output of the function is not a pointcloud. Instead its output is used by `VectorsToPointCloud`. I think it would make sense to rename this to `ToVectors`